lib/ex_aws/dynamo/util.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ExAws.Dynamo.Util do
+  @moduledoc """
+  A collection of light weight utility functions
+  """
+
+  @doc "Table exists"
+  @spec table_exists?(table_name :: String.t) :: boolean
+  def table_exists?(table_name) do
+    try do
+      ExAws.Dynamo.describe_table(table_name) |> ExAws.request!
+      true
+    rescue
+      ExAws.Error -> false
+    end
```

Comment:
Yeah, this is how I was thinking you could roll it - but I'd suggest reserving `try`/`rescue` for error handling situations, or where there are no other options.  Although `request!` is widely used in this repo, `ex_aws` actually just has a "normal" `request` method as well, which would allow you to write:

```
case ExAws.Dynamo.describe_table(table_name) |> ExAws.request do
  {:error, _} -> false
  _           -> true
end
```

Nothing wrong with what you wrote, but I think this would be more typical.